frontend/pages/beta/create-new-model.tsx

```diff
@@ -0,0 +1,97 @@
+import Box from '@mui/material/Box'
+import Button from '@mui/material/Button'
+import Card from '@mui/material/Card'
+import Divider from '@mui/material/Divider'
+import FormControlLabel from '@mui/material/FormControlLabel'
+import Radio from '@mui/material/Radio'
+import RadioGroup from '@mui/material/RadioGroup'
+import Stack from '@mui/material/Stack'
+import TextField from '@mui/material/TextField'
+import Typography from '@mui/material/Typography'
+import { useState } from 'react'
+
+import Wrapper from '../../src/Wrapper'
+
+type NewModelData = {
+  teamName: string
+  modelName: string
+  description: string
+  accessibility: 'public' | 'private'
+}
+
+export default function CreateNewModelWizard() {
```

Comment:
The URL for this page should probably be something simple like `/model/new` or `/model/create`.  The name might want to match with that.